src/main/java/com/fasterxml/jackson/databind/AnnotationIntrospector.java

```diff
@@ -719,6 +719,9 @@ public Boolean hasAnyGetter(Annotated a) {
         return names;
     }
 
+
```

Comment:
Javadoc would be great here. Also, since I plan on merging this in 2.11 branch (API add), adding "@since 2.11" there would make sense.